packages/tsconfig-reference/scripts/schema/result/schema.json

```diff
@@ -834,12 +840,13 @@
                       "esnext.decorators",
                       "esnext.object",
                       "esnext.regexp",
+                      "esnext.iterator",
                       "decorators",
                       "decorators.legacy"
                     ]
                   },
                   {
```

Comment:
Just for my understanding, could you tell me what these pattern changes mean?

Reply:
These options are case insensitive, so this is just the absolutely riduclous regex to verify the union of a bunch of strings

Reply:
In a better world, the regex syntax would accept syntax like `(?i)`, but JS doesn't have that syntax, it puts it _outside_ the regex expression and a load of consuming tools do not parse out strings like `"/blah/i"`